libc/src/unistd/CMakeLists.txt

```diff
@@ -160,6 +167,20 @@ add_entrypoint_object(
     .${LIBC_TARGET_OS}.lseek
 )
 
+add_entrypoint_object(
+  pathconf
+  ALIAS
+  DEPENDS
+    .${LIBC_TARGET_OS}.pathconf
+)
+
+add_object_library(
```

Comment:
You can remove line 177-182 as a whole. `pathconf_utils` is likely to be linux only here as you check special FS magic in those files. 

Instead of using `libc.src.unistd.pathconf_utils` in `DEPENDS`, just using `libc.src.unistd.linux.pathconf_utils`

Reply:
I dont really understand what you mean by this? Can you elaborate ?

Reply:
I meant you can move `libc.src.unistd.pathconf_utils` to `libc.src.unistd.linux.pathconf_utils` since `pathconf_utils` is `linux`-specific.